examples/metadata_demo.ipynb

```diff
@@ -64,37 +64,42 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
     "config = Configuration()\n",
-    "config.host = INGRESS"
+    "config.host = f\"{INGRESS}/seldon-deploy/api/v1alpha1\""
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# # auth if needed\n",
-    "# config.oidc_server = f\"{INGRESS}/auth/realms/deploy-realm\"\n",
-    "# config.oidc_client_id = \"sd-api\"\n",
+    "# auth if needed\n",
+    "config.oidc_server = f\"{INGRESS}/auth/realms/deploy-realm\"\n",
+    "config.oidc_client_id = \"sd-api\"\n",
+    "config.oidc_client_secret = \"sd-api-secret\"\n",
+    "config.auth_method = \"client_credentials\"\n",
+    "\n",
+    "# Alternativey set instead of the \"client_credentials\" flow\n",
     "# config.username = \"admin@seldon.io\"\n",
     "# config.password = \"12341234\"\n",
+    "# config.auth_method = \"password\"\n",
     "\n",
-    "# auth = OIDCAuthenticator(config)\n",
-    "# config.access_token = auth.authenticate()"
+    "auth = OIDCAuthenticator(config)\n",
+    "config.access_token = auth.authenticate()"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
-    "api_client = ApiClient(config,auth)\n",
+    "api_client = ApiClient(config)\n",
```

Comment:
Does this work with latest SDK? I thought you needed
```
api_client = ApiClient(configuration=config,authenticator=auth)
```

Reply:
Now you make me second guess myself. I just installed master and tested that...

Reply:
So with this I think token will get refreshed in case it expires. Didn't confirmed by looking into the code but that is the behaviour I just observed.

Reply:
Fixed now.